### PR TITLE
Fixes Medical Lamp & Desc Fixes

### PR DESCRIPTION
**Pull request by Technobug14**
## About The Pull Request
Medical lamps weren't working and would break into a TG floodlight and a broken TG light tube. A surgical mask bandana subtype had no description, so it inherited the black bandana's, fixed this. The US army helmet had some weird wording, changed it to sound better, imo!

## Why It's Good For The Game
WHOEVER MADE MEDICAL LAMPS A TG FLOODLIGHT SUBTYPE NEEDS TO BE PUT AGAINST A WALL AND SHOT

## Changelog

:cl:
fix: medical lamps :)
fix: surgical mask desc
tweak: army helmet desc
/:cl:

### CODE REVIEW

File: mojave/items/clothing/head/helmet.dm [hunk 1]
```diff
@@ -332,7 +332,7 @@
 
 /obj/item/clothing/head/helmet/ms13/army
 	name = "army helmet"
-	desc = "An old pre-war helmet, even by their standards. Seems to have the addition of quite a bit of ballistic padding on the inside, likely to try and modernize it."
```
Comment (ProfessorPopoff) [inline mojave/items/clothing/head/helmet.dm:335]: The only thing I'd have removed was 'even by their standards' and just went with the older vague initial description

File: mojave/structures/lighting/lamps.dm [hunk 2]
```diff
@@ -187,3 +187,37 @@
...
+	name = "medical lamp"
+	desc = "A once sterile lamp used in medical areas, good to get a view of your patient."
+	icon = 'mojave/icons/structure/medical.dmi'
+	icon_state = "medlamp"
+	density = TRUE
+	max_integrity = 250
+	anchored = TRUE
+	light_range = 4
+
+/obj/structure/ms13/lamp/attack_hand(mob/living/user, list/modifiers)
```
Comment (ProfessorPopoff) [inline mojave/structures/lighting/lamps.dm]: I don't think you intended to path it like this.

**Comment by Jeff11111** [comment]
where's the sounding though?

**Comment by Technobug14** [comment]
Fixed the bad path, and changed the desc again. I think it's better now, I agree keeping it more vague helps.

**Comment by ProfessorPopoff** [comment]
Yeah... sure

**Pull request merged**